_data/home.yml

```diff
@@ -31,7 +31,7 @@ cards:
       url: "/docs/user-stories"
   - title: "Contribute"
     icon: "fas fa-hands-helping"
-    description: "Powsybl is open source and part of the <a href='https://www.lfenergy.org/'>Linux Fundation Energy</a>. Help us to build the next generation power system tool!"
```

Comment:
Are you sure ?

Reply:
The "to" is actually optional with help, and mostly not used (https://ell.stackexchange.com/questions/47221/help-to-do-something-or-help-do-something).